ledger/participant-integration-api/src/main/scala/platform/index/IndexServiceOwner.scala

```diff
@@ -178,9 +178,9 @@ final class IndexServiceOwner(
       participantId = participantId,
       ledgerEndCache = ledgerEndCache,
       stringInterning = stringInterning,
-      completionsMaxPayloadsPerPayloadsPage = config.completionsMaxPayloadsPerPayloadsPage,
-      transactionsFlatStreamReaderConfig = config.transactionsFlatStreamReaderConfig,
-      transactionsTreeStreamReaderConfig = config.transactionsTreeStreamReaderConfig,
+      completionsPageSize = config.completionsPageSize,
+      transactionsFlatStreamsConfig = config.transactionsFlatStreams,
```

Comment:
Since we are already in the `Config` scope, we can drop the `Config` ending no? Also I would avoid (more as a standard) the double plural: `transactionsFlatStreams` vs `transactionFlatStreams`

Reply:
I removed the double plural but left the `Config` suffix. I think it's a useful hint as to what this variable holds, even thought it might seem redundant in this particular location